riotfile.py

```diff
@@ -129,7 +132,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="appsec",
             pys=select_pys(),
-            command="pytest {cmdargs} tests/appsec",
+            command="pytest --ddtrace {cmdargs} tests/appsec",
```

Comment:
Are there benefits/downsides for running this locally as well? If not, perhaps we can pass this option only via the CircleCI config

Reply:
We can run it locally as well, but it may not work properly for changes that are not committed, so maybe it's worth it to try to run it only in CircleCI

Reply:
How about passing a `--no-ddtrace` option locally to disable it?

Reply:
> but it may not work properly for changes that are not committed

So the best benefit of ITR is in CI then? If so, personally I'd prefer to have the CLI option passed only in via the circleci config to minimise the impact on contributors wanting to run tests locally.

Reply:
There should be some benefit locally, too, though it requires `DD_API_KEY` and `DD_APPLICATION_KEY` env vars, and either setting up the service in on https://app.datadoghq.com/ci/settings/test-service to be enabled for the `DD_SERVICE` and `DD_ENV` default .

I think passing the CLI option through the CircleCI is the safe option but, on the other hand, it might be nice to get a bit of "forced" adoption.